### ISSUE TITLE
Not Gloss and GLUT 2.6 are incompatible

### ISSUE THREAD

**Issue by TomMD** [ghorn/not-gloss#9]
Notice there is now a function `lookAt` in GLUT, which breaks the build of not-gloss:
```
src/Play.hs:32:42:
    Ambiguous occurrence ‘lookAt’
    It could refer to either ‘Linear.lookAt’,
                             imported from ‘Linear’ at src/Play.hs:6:1-13
                             (and originally defined in ‘Linear.Projection’)
                          or ‘Graphics.UI.GLUT.lookAt’,
                             imported from ‘Graphics.UI.GLUT’ at src/Play.hs:10:1-64
                             (and originally defined in ‘OpenGL-2.10.0.0:Graphics.Rendering.OpenGL.GLU.Matrix’)
```

**Comment by ghorn** [comment]
I can take care of this, thanks for reporting

**Comment by ghorn** [comment]
fixed in 5c7d4b6352626e46be58c8b063ea832d15c399b5, i also added travis-ci
